template/.gitignore

```diff
@@ -1,6 +1,9 @@
 # Ignore configuration files that may contain sensitive information.
 local.*
 *.local
+project.local.yml
```

Comment:
I wonder if we should just replace these three lines with *.local.* ? Also, where do settings.local.yml and sites.local.yml come into play? I don't see them in this PR.